lib/package-managers/base-package-manager.js

```diff
@@ -11,11 +11,19 @@ exports.BasePackageManager = class {
   }
 
   run(command, args = [], workingDirectory = process.cwd()) {
+    const isWindows = process.platform === "win32";
+    let executable = this.getExecutablePath(workingDirectory);
+    let options = { stdio: "inherit", cwd: workingDirectory };
+    if (isWindows) {
+      executable = `"${executable}"`
```

Comment:
The safer way to quote a string is to use `JSON.stringify(executable)`.

Reply:
Also please add a brief comment to explain what it fixes.

Reply:
Done